views/result.hbs

```diff
@@ -0,0 +1,33 @@
+<section class="mt5">
```

Comment:
I'm guessing these classes are all just the same tachyons classes from our prototype? In which case I would get rid of them or make sure this is linked to the style sheet (add a <link> tag in the main.hbs layout) because at the moment these classes are not doing anything